tests/testsuite/kubevirtresource.go

```diff
@@ -198,8 +198,10 @@ func UpdateKubeVirtConfigValue(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
 	patch, err := strategicpatch.CreateTwoWayMergePatch(old, newJson, kv)
 	Expect(err).ToNot(HaveOccurred())
 
-	kv, err = virtClient.KubeVirt(kv.Namespace).Patch(kv.GetName(), types.MergePatchType, patch, &metav1.PatchOptions{})
-	Expect(err).ToNot(HaveOccurred())
+	Eventually(func() error {
+		_, err = virtClient.KubeVirt(kv.Namespace).Patch(kv.GetName(), types.MergePatchType, patch, &metav1.PatchOptions{})
+		return err
+	}, 30*time.Second, 1*time.Second).Should(BeNil())
```

Comment:
@assafad Can you elaborate on the reason we need it? I see that the test changes cluster roles only. How come this has something to do with the flake?

Reply:
I'm not 100% sure, @machadovilaca would know better.

Reply:
@enp0s3 since we removed the CRB from the pods, sometimes when this is executed the pods are not yet working properly and this call is not successful